app/experimenter/experiments/views.py

```diff
@@ -525,14 +525,14 @@ def form_valid(self, form):
         return response
 
     def form_invalid(self, form):
-        return redirect(
-            "{url}?normandy_id={value}".format(
-                url=reverse(
-                    "experiments-detail", kwargs={"slug": self.kwargs["slug"]}
-                ),
-                value=form.data["normandy_id"],
-            )
+        url = reverse(
+            "experiments-detail", kwargs={"slug": self.kwargs["slug"]}
         )
+        query_parameters = form.data.copy()
+        if query_parameters.__contains__("csrfmiddlewaretoken"):
```

Comment:
:O You know there's a more idiomatic way to know if something is `in` something else, right?  There's a keyword I have `in` mind, perhaps it could provide some `in`sight into what you're doing here, perhaps I'm `in`dicating something......

Reply:
oh yea duh.